bazel/README.md

```diff
@@ -127,6 +127,11 @@ for how to update or override dependencies.
     startup --output_base=C:/_eb
     ```
 
+    See [Using Bazel on Windows](https://docs.bazel.build/versions/master/windows.html) page for 
+    other common issues, such as 
```

Comment:
Is this sufficient, or should we empathically state that they need to follow these;

s/other common issues, such as/instructions to enable/ ...

and close the whole idea with;

..., and other common issues.

Reply:
other common issues is fine.